### PR TITLE
Statsd support

### PR DESCRIPTION
**Pull request by debonair**
Changed to statsd supported endpoints

### CODE REVIEW

**Comment by debonair** [comment]
@rcrichton please review, this goes with the core branch

File: app/scripts/services/rest.js [hunk 1]
```diff
@@ -34,8 +34,9 @@ angular.module('openhimWebui2App')
       Mediators: $resource( server + '/mediators/:urn', { urn: '@urn' }),
 
       // add the metric endpoints
-      Metrics: $resource( server + '/metrics/:type/:channelId', {}),
-      MetricsStatus: $resource( server + '/metrics/status', {}),
+      Metrics: $resource( server + '/stats/:type/:channelId', {}),
```
Comment (rcrichton) [inline app/scripts/services/rest.js]: Hasn't the endpoint been changed back to `/metrics/` on the statsd model branch?

**Comment by rcrichton** [comment]
@debonair Are there any changes that need to happen on the metrics pages to get them to use the new API endpoints? Also, see my comment above.

**Comment by debonair** [comment]
@rcrichton  Sorry I had forgotte to push the changes to the metrics branch, I have updated the code and tests, please review.

**Comment by rcrichton** [comment]
Awesome, do all the metrics on core work as they used to? Or does anything else need to change? If nothing else then :+1:

**Comment by debonair** [comment]
Nothing else needs to change

**Pull request merged**